account_usability/report/invoice_report.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="report_invoice_document" inherit_id="account.report_invoice_document">
+    <xpath expr="//div[@name='origin']" position="replace">
```

Comment:
replace ?

Reply:
Yes but replace by a div with same name and the same html tag inside, then completely secured because completely overridable